volumio/bin/firststart.sh

```diff
@@ -15,6 +15,11 @@ rm -v /etc/ssh/ssh_host_*
 echo "Generating SSH host keys"
 dpkg-reconfigure openssh-server
 
+echo "Ensuring hotspot startup is controlled only by wireless.js"
```

Comment:
This should be removed, and we should not enable those services in the build script

Reply:
Thanks for that. I found a better place to do this, not sure if the firstboot.sh change is needed at all now.